src/main/java/com/eppo/sdk/helpers/ConfigurationRequestor.java

```diff
@@ -5,39 +5,31 @@
 
 import lombok.extern.slf4j.Slf4j;
 
-import com.eppo.sdk.constants.Constants;
-import com.eppo.sdk.dto.ExperimentConfigurationResponse;
 import com.eppo.sdk.exception.InvalidApiKeyException;
-import com.eppo.sdk.exception.NetworkException;
 
 import java.net.http.HttpResponse;
-import java.net.http.HttpTimeoutException;
 import java.util.Optional;
 
-/**
- * Experiment Configuration Requestor Class
- */
 @Slf4j
-public class ExperimentConfigurationRequestor {
+public class ConfigurationRequestor<T> {
```

Comment:
Templetized so it can easily be used for both types of files--standard RAC & bandit parameters--that we're requesting